gc/base/standard/Scavenger.cpp

```diff
@@ -2181,7 +2181,9 @@ MM_Scavenger::scavengeRememberedSetOverflow(MM_EnvironmentStandard *env)
 MMINLINE void
 MM_Scavenger::flushRememberedSet(MM_EnvironmentStandard *env)
 {
-	MM_SublistFragment::flush((J9VMGC_SublistFragment*)&env->_scavengerRememberedSet);
+	if (0 != env->_scavengerRememberedSet.count) {
```

Comment:
If the count is 0, and not checked here, it will be checked in MM_SublistPool::incrementCount() and MM_SublistFragment::flush(...) will essentially be a no-op. So checking it here just seemed to be a reasonable way to skip a small snippet of dead (in this case) code. I'm not attached to this as either way I would expect no measurable performance gain/loss.